larray_editor/comparator.py

```diff
@@ -238,28 +199,17 @@ def setup_and_check(self, sessions, names, title='', colors='red-white-blue'):
         main_splitter.setCollapsible(1, False)
 
         layout.addWidget(main_splitter)
+        self.listwidget.setCurrentRow(0)
 
-        # Buttons configuration
-        btn_layout = QHBoxLayout()
-        btn_layout.addStretch()
-
-        buttons = QDialogButtonBox.Ok
-        bbox = QDialogButtonBox(buttons)
-        bbox.accepted.connect(self.accept)
-        btn_layout.addWidget(bbox)
-        layout.addLayout(btn_layout)
-
-        self.resize(800, 600)
-        self.setMinimumSize(400, 300)
+    def _setup_edit_menu(self, menu_bar):
```

Comment:
it's a bit strange/unusual to have to override a method to make it do nothing. I think it would be better if the abstract class had an empty _setup_edit_menu method and subclasses override it to add the functionality. You will probably need to add one more class in the inheritance hierarchy to avoid duplicating functionality in SessionComparator and MappingEditor.